.github/workflows/python-code-quality.yml

```diff
@@ -149,6 +150,22 @@ jobs:
           name: sphinx-grass
           path: sphinx-grass
           retention-days: 3
+
+      - name: Bandit Vulnerability Scan
+        run: |
+          bandit -c pyproject.toml -iii -r . -f sarif -o bandit.sarif --exit-zero
+
+      - name: Upload Bandit Scan Results
+        uses: actions/upload-artifact@v4
```

Comment:
Would you be able to pin like the clang-format workflow, including the comment at the end? Renovate keeps the has and the comment updated with the correct granularity.